### ISSUE TITLE
Python Solutions

### ISSUE THREAD

**Issue by HAKSOAT** [mailtokartik1/InterviewBit-Solutions#1]
Am I allowed to add Python solutions?

**Comment by mailtokartik1** [comment]
Yes please create a fork and create a PR for the same. My idea is to have a folder structure for different labguages.
